package.cjs.json

```diff
@@ -0,0 +1 @@
+{ "type": "commonjs" }
```

Comment:
@unional Can't we bundle file as `cjs/index.cjs`? So we can get ride of this file?